source/documentation/tools/upgrading.md

```diff
@@ -0,0 +1,38 @@
+# Upgrading RStudio
+
+When moving between RStudio releases there are some differences that require either user intervention or caution.
+
+## Why should I upgrade?
+
+Upgrading to the latest version of RStudio has a number of benefits including newer versions of R, improved packages management options.
+
+## 2.2.6 to 3.0.11
+
+Key Points:
+
+- Conda has been replaced with Renv for package management
+- [RStudio 1.4](https://rstudio.com/products/rstudio/#rstudio-server)
+- [R version 4.0.3](https://cran.r-project.org/doc/manuals/r-release/NEWS.html)
+
+As Conda has now been removed. You will need to install packages using Renv. See [package management](package-management.html#renv)
+
+### Preparation
+
+- Please undile your RStudio if it is currently idled
+- Select RStudio 3.0.11 from the drop down and click yes on the prompt to install
+
+### Notable Packages
+
+- DBTools
```

Comment:
Should this be a heading rather than a bullet point?

Reply:
Ah yes!